src/main/java/com/github/erizo/gradle/JcstressPlugin.java

```diff
@@ -262,6 +262,16 @@ public void execute(Task task1) {
         return jcstressTask;
     }
 
+    private void setMainClass(JcstressTask jcstressTask) {
+        if (GradleVersion.current().compareTo(GradleVersion.version("8.0")) >= 0) {
+            jcstressTask.getMainClass().set("org.openjdk.jcstress.Main");
+        } else {
+            jcstressTask.setMain("org.openjdk.jcstress.Main");
```

Comment:
Hmm, I think this may still cause issues if linking against the new Gradle ABI.

Thinking more about it, both the old and new should probably be set Reflectively, but up to you.

Reply:
Yup, there'll be issues when I switch to the Gradle 8 API.

I'll fix that with the reflection.

Reply:
Fixed